### ISSUE TITLE
Remove `UserDeleteKludgeAdmin`

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#2509]
Once `two-databases` is merged into `master`, `UserDeleteKludgeAdmin` can be trashed since it's a workaround for installations where KPI and KoBoCAT share a single Postgres database.

_Originally posted by @jnm in https://github.com/kobotoolbox/kpi/pull/2504_

**Comment by noliveleger** [comment]
Also the code raise a 500 error because the it calls Django methods with Django 1.8 signature (which changed since).
